FaunaDB.Client/Types/RefParser.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace FaunaDB.Types
+{
+    public static class RefParser
+    {
+        public static RefV Parse(Value value)
+        {
+            return value.GetOption(Field.At("@ref")).Match(
```

Comment:
That parser only parse 2.1 refs. The fact that it try to get a `@ref` first is because the nature of the new representation.